Jenkinsfile.nightly

```diff
@@ -110,6 +111,16 @@ pipeline {
                 }
             }
         }
+        stage('Nightly-Docker-Image') {
+            steps {
+                withDockerRegistry([ credentialsId: "NIGHTLY_DOCKER_HUB_CRED", url: "" ]) {
+                    sh 'make docker-image'
+                    sh 'docker tag cilium/cilium:latest cilium/nightly:latest'
+                    sh 'export ID=$(date +%Y-%m-%d)--$BUILD_NUMBER && docker tag cilium/nightly:latest cilium/nightly:$ID && docker push cilium/nightly:$ID'
```

Comment:
I don't really like this export here, what about something like this: 
```groovy
stage('Nightly-Docker-Image') {

    environment {
        TAG = "${new Date().format("yyyyMMdd")}-${BUILD_NUMBER}"
    }

    steps {
        withDockerRegistry([ credentialsId: "NIGHTLY_DOCKER_HUB_CRED", url: "" ]) {
            sh 'make docker-image'
            sh 'docker tag cilium/cilium:latest cilium/nightly:latest'
            sh 'docker tag cilium/nightly:latest cilium/nightly:${TAG}'
            sh 'docker push cilium/nightly:${TAG}'
            sh 'docker push cilium/nightly:latest'
        }
    }
}
```

Reply:
Makes sense